semconv/v1.27.0/generated_attribute_group.go

```diff
@@ -810,7 +810,7 @@ const (
 	// Stability: experimental
 	// Examples: 'arn:aws:lambda:REGION:ACCOUNT_ID:function:my-function', '//run.googl
 	// eapis.com/projects/PROJECT_ID/locations/LOCATION_ID/services/SERVICE_ID', '/sub
-	// scriptions/<SUBSCIPTION_GUID>/resourceGroups/<RG>/providers/Microsoft.Web/sites
+	// scriptions/<SUBSCRIPTION_GUID>/resourceGroups/<RG>/providers/Microsoft.Web/sites
```

Comment:
If you haven't already, I'd suggest reviewing the semantic conventions repo (https://github.com/open-telemetry/semantic-conventions) as the source of these generated files lives there

Reply:
I have indeed, about a month ago. https://github.com/open-telemetry/semantic-conventions/pull/1379